tests/Shared/VerifyExtensions.cs

```diff
@@ -5,6 +5,9 @@
 
 internal static class VerifyExtensions
 {
+    /// <summary>
+    /// Sets the directory for the Verify call in a way it's also compatible with Helix.
+    /// </summary>
     public static SettingsTask UseHelixAwareDirectory(this SettingsTask settings, string directory = "Snapshots")
```

Comment:
Is this still used somewhere?

Reply:
It's not.

Didn't want to remove it yet to keep the knowledge of how it's done.